### DISCUSSION TITLE
Issues with apt repository

### DISCUSSION THREAD

**Discussion by jkuester** [apache/couchdb#5012] [Q&A] [answered]
Seems like something is wrong with https://apache.jfrog.io/artifactory/couchdb-deb/.  This is all I see when I visit that page:

![image](https://github.com/apache/couchdb/assets/8441903/df44ce7c-8fdf-42d9-91c2-540cd64c5306)

I double-checked [the docs](https://docs.couchdb.org/en/stable/install/unix.html#enabling-the-apache-couchdb-package-repository) and that still seems to be the correct repository. Or am I missing something here?

This is the observed error when trying to do an `apt-get update` with the couchdb-deb repo configured:

```
Failed to fetch https://apache.jfrog.io/artifactory/couchdb-deb/dists/bullseye/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
The repository 'https://apache.jfrog.io/artifactory/couchdb-deb bullseye InRelease' is not signed.
```

**Comment by nickva** [comment, marked as answer]
Thanks for your report @jkuester. Please see https://github.com/apache/couchdb/issues/5010
